crowbar_framework/lib/crowbar/conduit_resolver.rb

```diff
@@ -106,7 +106,7 @@ def if_speed_remap
 
         sorted_ifs.each do |intf|
           speeds = cr_ohai_network[intf]["speeds"]
-          speeds = ["1g"] unless speeds # legacy object support
+          speeds = ["1g"] if speeds.nil? || speeds.empty? # legacy object support
```

Comment:
doesn't work from irb:

irb(main):003:0> [].blank?
NoMethodError: undefined method `blank?' for []:Array
        from (irb):3
        from /usr/bin/irb:11:in `<main>'

is that really supposed to work in crowbar_framework?